src/about.js

```diff
@@ -25,32 +25,38 @@ ${getValaVersion()}
 ${getBlueprintVersion()}
 `.trim();
 
-  const dialog = new Gtk.AboutDialog({
-    application,
-    authors: ["Sonny Piers https://sonny.re"],
-    comments: _("Learn and prototype with\nGNOME technologies"),
+  const dialog = new Adw.AboutWindow({
+    application_name: "Workbench",
+    developer_name: "Sonny Piers",
     copyright: "Copyright 2022 Sonny Piers",
     license_type: Gtk.License.GPL_3_0_ONLY,
     version,
     transient_for: application.get_active_window(),
     modal: true,
     website: "https://workbench.sonny.re",
-    logo_icon_name: "re.sonny.Workbench",
+    application_icon: "re.sonny.Workbench",
+    issue_url: "https://github.com/sonnyp/Workbench/issues",
     // TRANSLATORS: eg. 'Translator Name <your.email@domain.com>' or 'Translator Name https://website.example'
     translator_credits: _("translator-credits"),
-    system_information,
+    debug_info: system_information,
+    developers: [
+      "Sonny Piers https://sonny.re",
+      "Lorenz Wildberg https://gitlab.gnome.org/lwildberg",
+    ],
+    designers: ["Tobias Bernard <tbernard@gnome.org>"],
   });
-  dialog.add_credit_section("Contributors", [
-    "Lorenz Wildberg https://gitlab.gnome.org/lwildberg",
-    "Tobias Bernard <tbernard@gnome.org>",
+
+  dialog.add_credit_section(_("Contributors"), [
     "Ben Foote http://www.bengineeri.ng",
+    "Hari Rana https://github.com/TheEvilSkeleton",
```

Comment:
@TheEvilSkeleton are you happy with this? feel free to suggest something else

Reply:
Is `Hari Rana (TheEvilSkeleton) https://theevilskeleton.gitlab.io` good for you? I like to have my alias there.